style.css

```diff
@@ -0,0 +1,1769 @@
+}
+
+.mitt {
+  height: 70px;
+}
+
+.event18 h3,
+.event17 h3 {
+  display: block;
+  height: 63.99px;
+  width: 112.11px;
+  margin-left: 18px;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  align-items: center;
+}
+
+.event17 a h3,
+.event18 a h3 {
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.separator {
+  width: 279px;
+  padding: 0 24px;
+  margin-left: 0;
+  margin-top: -10px;
+}
+
+.event-skill {
+  width: 295px;
+  height: 144px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: flex-start;
+  margin: 0;
+  font-weight: bold;
+  margin-left: -25px;
+}
+
+.contact-derek {
+  background-color: #6070ff;
+  border-radius: 50px 0 0 0;
+}
+
+.contact-derek h2 {
+  color: white;
+  margin-top: 40px;
+  text-align: center;
+}
+
+.contact-derek p {
+  color: #fff;
+  margin-left: 10px;
+  margin-right: 10px;
+  text-align: center;
+}
+
+#detail {
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 80%;
+}
+
+#btn {
+  border-radius: 8px;
+  background-color: white;
+  text-align: center;
+  color: #396df2;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 30%;
+  height: 50px;
+}
+
+hr.rounded {
+  border-top: 8px solid black;
+  width: 30%;
+  border-radius: 5px;
+}
+
+.form-button {
+  box-sizing: border-box;
+  align-items: center;
+  margin-left: 20px;
+  width: 12vw;
+  height: 9vh;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  color: #396df2;
+  transition: transform 0.4s ease-in-out;
+}
+
+#consection {
+  background: url(../images/contactformshapes.png);
+  background-repeat: no-repeat;
+  background-position: right;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  height: 800px;
+  flex: none;
+  order: 5;
+  flex-grow: 0;
+  align-content: center;
+}
+
+.contact {
+  margin: 100px 10px 0 10px;
+  list-style: none;
+}
+
+#h1 {
+  font-family: 'Poppins', sans-serif;
+  color: white;
+  margin-top: 40px;
+  text-align: center;
+  list-style: none;
+}
+
+.contact h1 {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  list-style: none;
+  justify-content: center;
+  text-align: center;
+}
+
+.contact p {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16 px;
+  line-height: 28px;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] As suggested by the previous reviewer please align your `paragraph` text in the center by applying the property `text-align:center` insteadd of applying it on `h1` heading tag  to make your design look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=eK11LcTtcp6JizSx-0)

|figma design|your design|
|---|----|
|<img width="215" alt="Screenshot 2023-06-13 at 9 53 58 AM" src="https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/44798044/cc7da1eb-e335-4ac7-afcd-d5a6cfb6dea5">|<img width="372" alt="Screenshot 2023-06-13 at 9 59 58 AM" src="https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/44798044/dd2b8d4e-392c-41fa-9ce7-58d7f3a57670">|

- [ ] [OPTIONAL] I would suggest you to fix the typo on your font-size as there should be no space between the pixel value and pixel unit like this `font-size: 16px;` instead of this `font-size: 16 px;`. Because of the typo the font-size is not taking any effect on your paragraph element. Also please increase the font size to at least `22px` and decrease the font weight to `200` to make your text look similar to [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=eK11LcTtcp6JizSx-0)

Reply:
done

![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/5b51eb58-d2e9-4988-a395-6d8939989450)